packages/components/src/styles/meta-information/meta-information.css

```diff
@@ -0,0 +1,32 @@
+.sd-meta-information {
```

Comment:
didnt we agree to change it to sd-meta or is there a follow up task for this change?

Reply:
It was a question yesterday and an agreement today. Definitely no follow-up to prevent breaking changes. :)

Commit follows.